xml/System/String.xml

```diff
@@ -4160,7 +4166,7 @@ Examples of instantiating strings:
 ## Remarks
  This method performs an ordinal (case-sensitive and culture-insensitive) comparison. The search begins at the first character position of this string and continues through the last character position.
 
- To determine whether a string contains a specified substring by using something other than ordinal comparison (such as culture-sensitive comparison, or ordinal case-insensitive comparison), you can create a custom method. The following example illustrates one such approach. It defines a <xref:System.String> extension method  that includes a <xref:System.StringComparison> parameter and indicates whether a string contains a substring when using the specified form of string comparison.
+ **.NET Framework only**: To determine whether a string contains a specified substring by using something other than ordinal comparison (such as culture-sensitive comparison, or ordinal case-insensitive comparison), you can create a custom method. The following example illustrates one such approach. It defines a <xref:System.String> extension method  that includes a <xref:System.StringComparison> parameter and indicates whether a string contains a substring when using the specified form of string comparison.
```

Comment:
Does writing .NET Framework only at the beginning make it seem like the ability exists only in .NET Framework and not in .NET core(when it actually does)?

Reply:
Not sure, I just copied this pattern. I defer to docs folks.